kmk/modules/modtap.py

```diff
@@ -2,7 +2,7 @@
 from kmk.modules.holdtap import HoldTap, HoldTapKeyMeta
 
 
-class ModTap(HoldTap):
+class HoldTap(HoldTap):
```

Comment:
This is the alias for `ModTap` to `HoldTap` and mustn't be renamed, or it'll introduce naming conficts. We will add a depercation notice here and remove at some point in the future.

Reply:
I have changed it back to `ModTap` and added a comment above the class.